templates/base.html

```diff
@@ -102,6 +107,33 @@
         </script>
         {% endblock script %}
         {% include '_includes/stat_counter.html' %}
+
+        {% if (( PHOTOS_LIGHTBOX == "True" ) and (article and article.photo_gallery)) %}
```

Comment:
Change it to

```
{% if PHOTOS_LIGHTBOX and article and article.photo_gallery %}
```

You don't need `true` or so many `()`.

Reply:
I did already in my latest code base (maybe yet not pushed)